contracts/dependencies/open-zeppelin/InitializableAdminUpgradeabilityProxy.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: agpl-3.0
```

Comment:
I don't think we're allowed to change the license if we fork. We might need to retain the license.